src/components/CSelectSearchable/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import Select, { MultiValue, ActionMeta, SingleValue } from 'react-select';
+
+import DropdownIndicator from '../CSelect/DropdownIndicator';
+import customStyles from '../CSelect/selectCustomStyles';
+import ValueContainer from './ValueContainer';
+import CTokenLabel from '../CTokenLabel';
+import SelectOption from './Option';
+
+export interface OptionType {
```

Comment:
Use BasicOptionType<string> in models (first, pull and merge master into this branch)